client/src/test/java/tds/session/SessionTest.java

```diff
@@ -19,4 +22,64 @@ public void shouldBeInstantiated() {
         assertThat(session.getType()).isEqualTo(0);
         assertThat(session.getStatus()).isEqualTo("closed");
     }
+
+    @Test
+    public void shouldBeOpen() {
```

Comment:
I don't have a strong opinion one way other the other; I've been using "should" as a carry-over from Ruby and .NET (although _much_ more from Ruby).